tests/build-test-snapd-snap

```diff
@@ -1,10 +1,47 @@
 #!/bin/bash -e
 
+help() {
+    echo "$0 [-h|--help] [--no-clean] [--clean-snapd-only]"
+    echo "builds test snapd using snapcraft"
+    echo
+    echo "options:"
+    echo "  --clean-snapd-only    - only clean the snapd part"
+    echo "  --no-clean            - do not clean anything"
```

Comment:
I think this is good but we should explain how it interacts with SNAPCRAFT_NO_CLEAN.

Reply:
done